sdcm/cluster_k8s/__init__.py

```diff
@@ -729,10 +729,13 @@ def deploy_minio_s3_backend(self):
         if not self.params.get('reuse_cluster'):
             LOGGER.info('Deploy minio s3-like backend server')
             self.kubectl(f"create namespace {MINIO_NAMESPACE}")
+            values = HelmValues({})
+            values.set('persistence.size', self.params.get("k8s_minio_storage_size") or "10Gi")
```

Comment:
Please, move default value ("10Gi") to the `test_default.yaml`

Reply:
Done